controllers/process.go

```diff
@@ -249,6 +308,7 @@ func refreshEnvoyFilter(instance *microservicev1alpha2.SmartLimiter, r *SmartLim
 // if configmap rate-limit-config not exist, ratelimit server will not running
 func refreshConfigMap(desc []*model.Descriptor, r *SmartLimiterReconciler, serviceLoc types.NamespacedName) {
 
+	// TODO, add switch
```

Comment:
what's this todo mean, explain more clearly

Reply:
it will add disableGlobalRatelimite swtich here